server/sonar-web/src/main/js/apps/permissions/project/components/PageHeader.js

```diff
@@ -35,11 +36,12 @@ type Props = {|
   loadHolders: () => void,
   loading: boolean
 |};
+*/
 
 export default class PageHeader extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  handleApplyTemplate = (e: Event & { target: HTMLButtonElement }) => {
+  handleApplyTemplate = (e /*: Event & { target: HTMLButtonElement } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)